README.md

```diff
@@ -59,13 +55,13 @@ The script will:
 - clone the repos locally, only copying the current version of the files
 - build a `rmi_pacta:<tag>` and `rmi_pacta:latest` docker image (where `<tag>` is the tag you provided, e.g. `0.1.14`). The image builds from the Dockerfile in this directory, which will
 
-  - use [`r-base:<version>`](https://hub.docker.com/_/r-base) as the base (where <version> is the R version specified in the Dockerfile)
+  - use [`rocker/r-ver:<version>`](https://hub.docker.com/r/rocker/r-ver/tags) as the base (where <version> is the R version specified in the Dockerfile)
   - install system dependencies
   - install latex and dependencies
   - install dependent TeX packages
   - install dependent R packages and system dependencies
   - copy in the freshly cloned repos
-  - install the appropriate PACTA R packages in the Docker image's R environment from the copied repos
+  - install the appropriate PACTA R packages in the Docker image's R environment from their respective GitHub repos
   - make some necessary permissions changes
   - set the `build_version` environment variable with the specified tag
```

Comment:
NIT: Is it necessary/ desirable to re-state everything that the `Dockerfile` does here? 
It seems likely that this will fall out of date, and I'm not sure how much value it adds if people can just look at the `Dockerfile`?

Reply:
possibly not, but I'll leave that for another day (honestly hoping this repo gets archived before another update of the README is necessary).

Reply:
Sounds good! Just something to keep in mind for the next iteration then :-)